Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.11-alpine as builder
+
+# Install our build tools
+
+RUN apk add --update git make curl bash
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
I've never been a big fan of piping scripts from the internet into bash. How would you feel about just building from source instead ? We won't need curl and bash above this way either.

```go
go get -u github.com/golang/dep/cmd/dep
```

Reply:
That's much cleaner. I've updated the Dockerfile accordingly.

I've left bash in there  - it's also used to run `build/build.sh`
I don't know if there is any bash-specific syntax in there.

Reply:
Good catch. In that case let's put `bash` back at the top with other `apt add` to avoid extra layer being created after checking in the sources, and LGTM !

Reply:
Done